doc/api/next_api_changes/2018-11-20-AL.rst

```diff
@@ -0,0 +1,4 @@
+Deprecations
+````````````
+
+The ``Path.has_nonfinite`` attribute is deprecated.
```

Comment:
IMHO we should state the alternative here as well.

Reply:
done